Caltech-IPAC/firefly
# Firefly-179: Fixed right-east align and align w/rotation issues

Firefly-179: Fixed right-east align and align w/rotation issues
 
  - right-east image align now work
   - fixed issue with aligning to rotated images
   - fixed align left-east with right-east image
   - clean up of rotation dialog: (fixes some issues in FIREFLY-146)
        - fixed multiple issues with rotation dialog and improved it.
        - now does immediate results
        - operates on active image.
        - has a north up button
        - show active images's rotation
        - Added slider
        - rotation dialog hides when all image removed
 
https://jira.ipac.caltech.edu/browse/FIREFLY-179

_To Test:_

 https://irsawebdev9.ipac.caltech.edu/firefly-179-wcs-match-bug/firefly/

  - load two images at same target (i.e. 2mass, wise), rotate one then match the other to it. Check that they scroll around together
  - load two east-right images (firefly_test_data/east-right-images)
  - you should be able to match them
  - load two east-right images, rotate one, then match
  - load a each right image from `firefly_test_data/east-right-images` then load a wise at 
`3h32m02.85s, -27d26m20.7s`, 792 arcsec.  try matching.
   - bring up the updated rotation dialog and play with it after loading a set of images.
